cfpb/cfgov-refresh
# Add last_edited field to Ask CFPB export CSV

For more easily figuring out which pages might need updating sooner, rather than later.

Addresses https://GHE/CFGOV/platform/issues/3781


## Additions

- Adds `last_edited` field to the Ask CFPB export CSV


## Testing

1. Pull branch
1. Go to http://localhost:8000/admin/export-ask and run the export
1. Open the CSV and see the new LastEdited column


## Checklist
